dev/sparktestsupport/utils.py

```diff
@@ -84,7 +84,7 @@ def identify_changed_files_from_git_commits(patch_sha, target_branch=None, targe
         ["git", "diff", "--name-only", patch_sha, diff_target], universal_newlines=True
     )
     # Remove any empty strings
-    return [f for f in raw_output.split("\n") if f]
+    return [f for f in raw_output.split("\n") if f and not f.endswith("README.md")]
```

Comment:
Should we maybe fix it in L48 `if filename in ("appveyor.yml",):` at `determine_modules_for_files` above?

Reply:
Oh, right. I forgot that. Let me move this there.